test/expected.json

```diff
@@ -49,6 +49,7 @@
         "water-huge": 0.0011,
         "water-huge2": 0.0028,
         "bad-hole": 0.019,
+        "issue16": 4e-16,
```

Comment:
I had to add this to make the tests pass. I don't understand well enough why this is needed, but this seems to be reasonable?

Reply:
Can you check [this test case](https://github.com/mapbox/earcut/blob/master/test/fixtures/issue16.json) visually and see how it looks before/after? Just to make sure. The original issue for it is #16.

Reply:
Looks like just one triangle flipped for this test case.

Before | After
--- | ---
![image](https://user-images.githubusercontent.com/30893333/124890637-890a0f80-dfd8-11eb-90d2-a9c526af043a.png) | ![image](https://user-images.githubusercontent.com/30893333/124890669-90c9b400-dfd8-11eb-8e21-1aa42248500c.png)